src/invidious/metrics.cr

```diff
@@ -0,0 +1,48 @@
+# Module containing an optional Kemal handler, which can be used
+# to collect metrics about the usage of various Invidious routes.
+module Metrics
+  record MetricLabels, request_method : String, request_route : String, response_code : Int32
+
+  # Counts how many a given route was used
+  REQUEST_COUNTERS              = Hash(MetricLabels, Int64).new
+
+  # Counts how much time was used to handle requests to each route
+  REQUEST_DURATION_SECONDS_SUMS = Hash(MetricLabels, Float32).new
```

Comment:
If you could use [class variables](https://crystal-lang.org/reference/1.9/syntax_and_semantics/class_variables.html) here, that'd be better.
I'd like to avoid misused constants as much as possible.

See the following for an example use case:
https://play.crystal-lang.org/#/r/fgvi

Reply:
Using class variables now, thanks for the example!